clarify description (#1008)
mate-academy/py-game-models: README.md

Each skill has:
- `name` - a *unique* char field with a maximum length of 255 characters.
- `bonus` - a char field with a maximum length of 255 characters. 

> This field describes what kind of bonus players can get from it. With other words, it is description of bonus.

Reviewer: ```suggestion
This field describes what kind of bonus players can get from it. In other words, this is a description of the bonus.
```

- `race` - a foreign key that points to the `Race` model. It shows which race has the corresponding skill.

> **Please note:** skill must be deleted when the race is deleted.

Reviewer: ```suggestion
**Important Note:** The skill must be deleted when the race is deleted.
```

And finally, a `Player` model.
It should have the following fields:
- `nickname` - a *unique* char field with a maximum length of 255 characters.

> - `email` - an email field with a maximum length of 255 characters. It may not be unique.

Reviewer: ```suggestion
- `email` - an email field with a maximum length of 255 characters. It can be non-unique.
```

- `bio` - a CharField with a maximum length of 255 characters. 
It stores a short description provided by a user about himself/herself.
- `race` - a foreign key that points to the `Race` model and shows 
the race of the player.

> **Please note:** player must be deleted when the race is deleted.

Reviewer: ```suggestion
**Important Note:** The player must be deleted when the race is deleted.
```

## Second Task:

Implement function `main()` in `main.py` which will have the following logic:

Read data about players from `players.json` and add the corresponding entries to the database.
Note, that some guilds, races and skills are used for different players. Create only one
instance for each guild, race and skill, do not copy them.

**Note**: It would be very useful to use the 
[get_or_create()](https://docs.djangoproject.com/en/4.2/ref/models/querysets/#get-or-create) method here.

> We don't care about performance for this task, so we should be OK to query database to check whether row already exist.
> It's also no need to use `bulk_create` here to avoid complexity of this task.

Reviewer: ```suggestion
We don't prioritize performance for this task, so querying the database to check whether a row already exists is acceptable.
There’s no need to use `bulk_create` in this case, as it adds unnecessary complexity to the task.
```
